rsmtool/analyzer.py

```diff
@@ -529,16 +525,15 @@ def correlation_helper(df,
         df_target_partcors : pandas DataFrame
             Data frame containing Pearson's correlation coefficients for
             partial correlations between each feature and `target_variable`
-            after controlling for all other features. If include_length is
-            set to True, `length` will not be included into partial
+            after controlling for all other features. If ``include_length`` is
+            set to ``True``, `length` will not be included into partial
             correlation computation.
         df_target_partcors_no_length: pandas DataFrame
-            If `include_length` is set to `true`: Data frame containing
+            If ``include_length`` is set to ``True``: Data frame containing
             Pearson's correlation coefficients for partial correlations
             between each feature and `target_variable` after controlling
```

Comment:
Should this also have two ticks? I'm not certain of the context here.

```suggestion
            between each feature and ``target_variable`` after controlling
```